addon/testem.js

```diff
@@ -1,23 +0,0 @@
-'use strict';
```

Comment:
some additional cleanup here that was missed in the test-app extraction, or came back during one of my rebases